source/documentation/runbooks/internal/how-to-be-support.md.erb

```diff
@@ -33,26 +33,44 @@ Monitor this channel and respond to queries and requests as they come in. If you
 
 There are a number of email groups you should monitor for customer queries and requests. They are:
 
+TODO: Add links to the following
```

Comment:
If add the links to these groups, as they are private links, it will fail on the URL check in the tech-doc-publisher, and will need to be ignored in the post deployment url check [here](https://github.com/ministryofjustice/operations-engineering/blob/52a2e1c824f775b262ec439678288a9697d59f85/.github/workflows/publish.yml#L85)

Reply:
Added a regex for google groups.

Reply:
Cool if that works, the other links you have added are to private areas inside the accounts, not sure what the result will be when the URL test check them